tests/app-interface/docs/context.md

```diff
@@ -0,0 +1,126 @@
+# App-Interface
```

Comment:
Clean this up and include info on why this is here and how you build data.json. Nothing else